src/js/index.js

```diff
@@ -0,0 +1,102 @@
+// main actions  with keyboard will be there
+import createKeyboardView from './keyboardView';
+import getLocalLanguage from './getLocalLanguage';
+import keyboardPressedEvents from './keyboardPressedEvents';
+import changeToPressed from './changeToPressed';
+import changeToUnpressed from './changeToUnpressed';
+import keyboardUnpressedEvents from './keyboardUnpressedEvents';
+import writeSymbol from './writeSymbol';
+import mouseEvents from './mouseEvents';
+import moveCursor from './moveCursor';
+
+
+createKeyboardView();
+
+const changeRegister = () => {
+
+    document.querySelectorAll('.keyboard__button').forEach((button) => {
+        button.childNodes.forEach((element) => {
+            element.childNodes.forEach((item) => item.classList.toggle('on'))
+        });
+
+    });
+};
```

Comment:
combine that function with changeRegisterByCapsLock

Reply:
Done